wrf/utils.py

```diff
@@ -824,19 +824,30 @@ def combine_towers(fdir, restarts, simulation_start, fname, return_type='xarray'
     This will work with a pandas df or an xarray ds/da
     '''
     for rst,restart in enumerate(restarts):
+        if np.size(simulation_start) == 1:
+            sim_start = simulation_start
+        elif np.size(simulation_start) == np.size(restarts):
+            sim_start = simulation_start[rst]
+        else:
+            print('restarts and simulation_start are not equal')
```

Comment:
Please update the example in the docstring (line 819) to reflect this requirement